operator/ingress/v1/0000_50_dns-record.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
use v1

Reply:
When I changed this to v1, the CRD generator silently stops working for the file. Not sure why. I notice there are many other CRDs using v1beta1.... any ideas?